src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py

```diff
@@ -258,13 +373,21 @@ def acr_repository_delete(cmd,
                                "and all images under it?".format(repository), yes)
             path = '/v2/_acr/{}/repository'.format(repository)
         else:
+            logger.warning(
+                "This command is deprecated. The new command for this operation "
+                "is 'az acr repository untag --name %s --image %s:%s'.",
```

Comment:
This is good.  Looks like it still works but points them in the right direction.  I maybe just didn't understand the earlier logic?